client/src/app/shared/video/abstract-video-list.scss

```diff
@@ -1,6 +1,45 @@
+@import '_variables';
 @import '_mixins';
 @import '_miniature';
 
+.avl-item {
+  // variables
+  $gutter: 10px;
+
+  // local mixin
+  @mixin avl-miniatures($n) {
+    --nb_miniatures: #{$n};
+
+    &:nth-of-type(#{$n}n) {
+      margin-right: unset;
+    }
+  }
+
+  // .avl-item
+  @media screen and (max-width: 1000px) {
+    @include avl-miniatures(2);
+  }
+
+  @media screen and (min-width: 1001px) and (max-width: 1200px) {
+    @include avl-miniatures(3)
+  }
+
+  @media screen and (min-width: 1201px) and (max-width: 1600px) {
+    @include avl-miniatures(4);
+  }
+
+  @media screen and (min-width: 1601px) and (max-width: 1800px) {
+    @include avl-miniatures(5);
+  }
+
+  @media screen and (min-width: 1801px) {
+    @include avl-miniatures(6);
+  }
+
+  margin-right: #{$gutter};
+  width: calc(100% / var(--nb_miniatures) - (#{$gutter} * (var(--nb_miniatures) - 1) / var(--nb_miniatures)));
```

Comment:
I think you should set a max width, because thumbnails are quite small

Reply:
I ended up switching to a grid-based display, which is more maintainable and guarantees a max-size based on the minimum size specified.